src/components/BareMetalHosts/tests/BaremetalHostStatus.test.js

```diff
@@ -5,16 +5,13 @@ import { BaremetalHostStatus } from '..';
 
 import BaremetalHostStatusFixture from '../fixtures/BaremetalHostStatus.fixture';
 
-const testBaremetalHostStatus = () => <BaremetalHostStatus {...BaremetalHostStatusFixture[0].props} />;
-const testBaremetalHostStatusSuccess = () => <BaremetalHostStatus {...BaremetalHostStatusFixture[1].props} />;
+const getComponentFunction = props => () => <BaremetalHostStatus {...props} />;
 
 describe('<BaremetalHostStatus />', () => {
-  it('renders a generic status message', () => {
-    const component = shallow(testBaremetalHostStatus());
-    expect(component).toMatchSnapshot();
-  });
-  it('renders a generic success message', () => {
-    const component = shallow(testBaremetalHostStatusSuccess());
-    expect(component).toMatchSnapshot();
+  it('renders the correct subcomponent', () => {
```

Comment:
it should be inside the forEach for easier debugging  + please also include the fixture index in `it` message + optionally the status

Reply:
Good point, thanks. I made the message more descriptive by adding the status but omitted the index because that would make the message ugly and wouldn't add to its meaning.

Reply:
index add uniqueness to the test case. This will break once you check for the same statuses in multiple fixtures (different conditions - same status)